test/check-exercises.py

```diff
@@ -11,6 +11,9 @@
 import tempfile
 import json
 
+# Allow high-performance tests to be skipped
+ALLOW_SKIP = ['largest-series-product']
```

Comment:
I'm happy with this solution - if we decide to do something different in the future, this list will make it easy to know where the changes need to be made anyway.